etcdutl/snapshot/util.go

```diff
@@ -16,16 +16,20 @@ package snapshot
 
 import (
 	"encoding/binary"
+	"fmt"
 )
 
 type revision struct {
 	main int64
 	sub  int64
 }
 
-func bytesToRev(bytes []byte) revision {
+func bytesToRev(bytes []byte) (revision, error) {
+	if len(bytes) != 17 {
+		return revision{}, fmt.Errorf("invalid revision size: %d", len(bytes))
```

Comment:
nit: `invalid revision size: %d` -> `invalid revision size: %d, expected size is 17 (2 * size(int64) + delimiter)`

Reply:
Done.